FirebaseVertexAI/Sources/GenerativeAIService.swift

```diff
@@ -267,20 +267,20 @@ struct GenerativeAIService {
     if error.isFirebaseMLServiceDisabledError() {
       VertexLog.error(code: .vertexAIInFirebaseAPIDisabled, """
       The Vertex AI for Firebase SDK requires the Firebase ML API `firebaseml.googleapis.com` to \
-      be enabled for your project. Get started in the Firebase Console \
-      (https://console.firebase.google.com/project/\(projectID)/genai/vertex) or verify that the \
-      API is enabled in the Google Cloud Console \
-      (https://console.developers.google.com/apis/api/firebaseml.googleapis.com/overview?project=\
-      \(projectID)).
+      be enabled for your project. Enable it by visiting the the Firebase Console at
+      https://console.firebase.google.com/project/\(projectID)/genai/vertex then retry. If you
+      enabled this API recently, wait a few minutes for the action to propagate to our systems and
+      retry.
       """)
     }
 
     if error.isVertexAIInFirebaseServiceDisabledError() {
       VertexLog.error(code: .vertexAIInFirebaseAPIDisabled, """
       The Vertex AI for Firebase SDK requires the Firebase Vertex AI API \
-      `firebasevertexai.googleapis.com` to be enabled for your project. Get started by visiting \
-      the Firebase Console at: \
-      https://console.firebase.google.com/project/\(projectID)/genai/vertex
+      `firebasevertexai.googleapis.com` to be enabled for your project. Enable it by visiting the \
+      the Firebase Console at https://console.firebase.google.com/project/\(projectID)/genai/vertex
```

Comment:
```suggestion
      the Firebase Console at https://console.firebase.google.com/project/_/genai/ and clicking "Get started".
```

Reply:
Updated the wording. Note: I kept `\(projectID)` instead of `_` in the URL since we have access to the exact project ID where this log message is emitted.